source/installation_and_configuration/authentication/overview.rst

```diff
@@ -26,13 +26,13 @@ You can choose from the following authentication drivers to access OpenNebula fr
 
 By default, any authentication driver configured to work with OpenNebula can be used out-of-the-box with Sunstone. Additionally you can add a TLS-proxy to secure the Sunstone. See:
 
-- :ref:`Sunstone Authentication <sunstone>`
+- :ref:`Sunstone Authentication <ruby_sunstone>`
```

Comment:
i think is better to remove all Ruby Sunstone references. We'll move the standalone guides to Legacy section, and remove all references to Ruby Sunstone in the main doc